.circleci/config.yml

```diff
@@ -199,6 +205,7 @@ jobs:
             mv scripts/datadog-synthetics-preview.js /tmp/workspace/scripts/datadog-synthetics-preview.js
             mv scripts/datadog-synthetics-ci.sh /tmp/workspace/scripts/datadog-synthetics-ci.sh
             mv scripts/postinstall.sh /tmp/workspace/scripts/postinstall.sh
+            mv scripts/process_images.sh /tmp/workspace/scripts/process_images.sh
```

Comment:
moving script for compressing images to `/tmp/workspace/` for ssh

Reply:
nit: can we keep rename it to process-image.sh or rename the other files to follow this naming convention?